feature/src/main/resources/feature.xml

```diff
@@ -52,38 +52,56 @@ limitations under the License.
         <bundle>mvn:org.apache.jclouds.api/atmos/${jclouds.version}</bundle>
```

Comment:
Hmm indeed not clear at all :)
I was referring to the the [root element namespace](https://github.com/andreaturli/jclouds-karaf/blob/44bec200d1c6241c3b042217df78e09529ef61a2/feature/src/main/resources/feature.xml#L18) and the `resolver` attribute at each element.